src/amplitude-wrapper.js

```diff
@@ -154,6 +154,15 @@ var amplitudeUserAgentEnrichmentPlugin=function(i){"use strict";var e=function()
   var init = function(client, args) {
       const argsLength = args.length;
       const configuration = args[argsLength - 1];
+
+      const excludeReferrers = [
+        ...configuration.defaultTracking.attribution.excludeReferrersText || [],
+        ...configuration.defaultTracking.attribution.excludeReferrersRegex?.map(item => new RegExp(item)) || []
+      ];
+      delete configuration.defaultTracking.attribution.excludeReferrersText;
```

Comment:
Is this passed by reference? Could this delete the template setting?

Reply:
It's passed by reference, it will delete `excludeReferrersText` in `config` and also `args[argsLength - 1]`. Do you have any concern over it?

Reply:
Discuss offline. it won't change the template UI input.